arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -301,7 +303,9 @@ void RocksDBEngine::collectOptions(std::shared_ptr<options::ProgramOptions> opti
 
   options->addOption("--rocksdb.throttle", "enable write-throttling",
                      new BooleanParameter(&_useThrottle));
-
+  
+  // TODO: move this to enterprise code and add the enterprise-only flag to it, e.g.
+  // arangodb::options::makeFlags(arangodb::options::Flags::Enterprise)
```

Comment:
Ahh, I remember why this is in community.  The code for sha creation was more easily incorporated in the community version.  Therefore, the flag was created to allow easier testing of community code.  How about we just make this a hidden flag or enterprise flag, but still leave it here?  There is some overhead to creating the sha files.  So an enterprise customer never using hot backup might want to disable.  This last item is why there is a flag offered.

Reply:
Ok. Then adding the `Enterprise` flag will do.